### ISSUE TITLE
Native modules with dependencies in Android

### ISSUE THREAD

**Issue by gmusick** [electrode-io/electrode-native#885]
Hi electrode-native team,

I have a native dependency (https://github.com/corymsmith/react-native-fabric) that I'd like to include in my app. For Android, one of the dependencies of that native dependency needs to be compiled with the `transitive = true` flag in the `build.gradle` file:

```
compile('com.crashlytics.sdk.android:crashlytics:2.9.2@aar') {
  transitive = true;
}
```

The current `dependencies` property in the manifest override only takes a string and doesn't seem to allow for something like this. Is this possible?

Thanks.

**Comment by krunalsshah** [comment]
@gmusick check this out https://github.com/electrode-io/electrode-native-manifest/blob/master/plugins/ern_v0.17.0%2B/react-native-maps_v0.21.0%2B/config.json#L2

By having a manifest entry for native package & specify the configuration , you should be able to pull the native source transitively. You are most welcomed to create a PR in `electrode-native-manifest` repo. Let us know if there is any way we can help. thanks

**Comment by krunalsshah** [comment]
@gmusick https://github.com/electrode-io/electrode-native/pull/892 

this may be related problem you are facing?

**Comment by belemaire** [comment]
@gmusick 
You are correct, we don't support adding the `transitive` flag.
This is something we should support. Targeting v0.24.0 release.

**Comment by belemaire** [comment]
@gmusick 
Just openeded https://github.com/electrode-io/electrode-native/pull/897 to take care of this. It'll be shipped in Electrode Native 0.24.0 in the upcoming days.
Once released, you'll be able to specify transitive dependencies by adding `transitive:` prefix in the list of dependencies to inject for the plugin.

Instead of doing this in the plugin configuration :

```gradle
"dependencies": [
  "com.crashlytics.sdk.android:crashlytics:2.9.2@aar"
]
```

which will inject `compile 'com.crashlytics.sdk.android:crashlytics:2.9.2@aar'` in the Android Container.

You'll be able to do this :

```
"dependencies": [
  "transitive:com.crashlytics.sdk.android:crashlytics:2.9.2@aar"
]
```

which will inject `compile('com.crashlytics.sdk.android:crashlytics:2.9.2@aar') { transitive = true }` in the Android Container.

Thanks for reporting this one :+1:
